saleor/product/templatetags/sort_menu.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+
+from ..filters import DEFAULT_SORT
+
+register = template.Library()
+
+
+@register.inclusion_tag('category/_sort_menu.html', takes_context=True)
+def sort_menu(context, attributes):
+    context['sort_by'] = \
+        context['request'].GET.get('sort_by', DEFAULT_SORT).strip('-')
+    context['sort_by_choices'] = attributes
+    context['arrow_down'] = \
+        (context['request'].GET.get('sort_by', DEFAULT_SORT).startswith('-'))
+    return context
```

Comment:
Again, instead of returning whole context you received as first argument, you should construct your own dictionary with data required by `_sort_menu.html` snippet.

Reply:
same as above